### ISSUE TITLE
Plugin can't be translated

### ISSUE THREAD

**Issue by dgsiegel** [verbb/smith#16]
**Description**

Even though static message translations are given, the plugin still shows the default strings "Copy", "Clone" and "Paste". I can't find those strings in JSON translations strings in the CP either.

**Steps to reproduce**
1. Add static translations to either `translations/de/app.php` or `translations/de/smith.php`: 

```
<?php

return [
  "Clone" => "Duplizieren",
  "Copy" => "Kopieren",
  "Paste" => "Einfügen",
];
```
2. Still find the old strings in the CP

**Additional info**
- Plugin version: 1.1.12
- Craft version: 3.6.10
- Multi-site: Yes

**Comment by engram-design** [comment]
Fixed in [1.1.13](https://github.com/verbb/smith/releases/tag/1.1.13). Please supply translations in your app.php file.

**Comment by dgsiegel** [comment]
Thanks for the quick fix!
